client/src/components/ManageArtist/UploadImage.tsx

```diff
@@ -52,7 +52,8 @@ export const UploadPrompt: React.FC<{
   width?: string;
   height?: string;
   rounded?: boolean;
-}> = ({ width, height, rounded }) => {
+  imagetypedescription?: string;
```

Comment:
For consistency's sake and readability, can you make this camelcase? `imageTypeDescription`